src/BankSystem/Main.java

```diff
@@ -1,85 +0,0 @@
-package BankSystem;
-
-import java.util.Scanner;
-
-public class Main {
-    public static void main(String[] args) {
-//        to run the attributes and methods of BankAccount
-//        Scanner input = new Scanner(System.in);
-//        System.out.println("Enter your name, please ");
-//        String clientName=input.next();
-
-//BankAccount class
-        BankAccount client=new BankAccount("1234567899","Shereen Saad",11.000);
-        System.out.println(client.getHolderName());
-        System.out.println(client.getAccountNumber());
-        System.out.println(client.getBalance());
-
-
-
-//        trying to use if condition with oop
-//        if ((client.getHolderName()).equals(clientName)){
-//            System.out.println("Your account number is "+client.getAccountNumber());
-//            System.out.println("Your balance is "+client.getBalance());
-//        }else {
-//            System.out.println("You are not client in this bank");
-//        }
-
-        System.out.println("=======================================");
-//Account
-        Account account=new Account("123456789",5000,"advanced");
-        System.out.println(account.getAccountNumber());
-        System.out.println(account.getBalance());
-        account.services();
-
-        System.out.println("=======================================");
```

Comment:
I recommend creating objects using the sub class instead of super, in order to emphasize the inheritance concept.